### PR TITLE
Kubectl add image pull secrets

### PR DESCRIPTION
**Pull request by minollo**
Needed across all components to support private registries.

### CODE REVIEW

File: kubectl-hlf/cmd/ca/create.go [hunk 1]
```diff
@@ -300,6 +311,7 @@ func newCreateCACmd(out io.Writer, errOut io.Writer) *cobra.Command {
 	f.StringArrayVarP(&c.caOpts.Hosts, "hosts", "", []string{}, "Hosts for Istio")
 	f.StringVarP(&c.caOpts.IngressGateway, "istio-ingressgateway", "", "ingressgateway", "Istio ingress gateway name")
 	f.IntVarP(&c.caOpts.IngressPort, "istio-port", "", 443, "Istio ingress port")
+	f.StringArrayVarP(&c.caOpts.ImagePullSecrets, "image-pull-secrets", "s", []string{}, "Image Pull Secrets for the CA Image")
```
Comment (adityajoshi12) [inline kubectl-hlf/cmd/ca/create.go:314]: `-s` is already used for storage class option, can you consider using different, variable for pull secret.

**Review by adityajoshi12 — COMMENTED** [review]
`-s` is already used for storage class option, can you consider using different variable for pull secret?

**Comment by minollo** [comment]
Sorry about that - I had just blindly used the same choice you made for external chaincodes. I have removed the shorthand setting, as this is a fairly advanced option anyway.

**Comment by minollo** [comment]
> `-s` is already used for storage class option, can you consider using different variable for pull secret?

Sorry about that - I had just blindly used the same choice you made for external chaincodes. I have removed the shorthand setting, as this is a fairly advanced option anyway.

**Comment by adityajoshi12** [comment]
Can you please also remove the shorthand from `kubectl-hlf/cmd/externalchaincode/create.go ` to make it consistent across the application

**Comment by minollo** [comment]
> Can you please also remove the shorthand from `kubectl-hlf/cmd/externalchaincode/create.go ` to make it consistent across the application

Sure; I have also fixed the singular to plural in external chaincode... it bothered me a bit.

**Review by adityajoshi12 — APPROVED** [review]
LGMT

**Review by dviejokfs — APPROVED** [review]

**Pull request merged**